content/intro-to-storybook/react/en/get-started.md

```diff
@@ -23,17 +23,18 @@ cd taskbox
 npx -p @storybook/cli sb init
 ```
 
-<div class="aside">
-With a recent update to React and CRA you'll need to downgrade <code>react-scripts</code> package to a previous version in order to run Storybook without any issues, version 3.4.3 will work perfectly. Also both the <code>React</code> and <code>ReactDOM</code> will require a downgrade to a previous version. Version <code>16.14.0</code> of both packages will work perfectly.
-<p>See the Yarn official documentation on how to <a href="https://yarnpkg.com/cli/add">add</a> and <a href="https://yarnpkg.com/cli/remove">remove</a> packages.</p>
-</div>
-
 <div class="aside">
 Throughout this version of the tutorial, we'll be using <code>yarn</code> to run the majority of our commands. 
 If you have Yarn installed, but prefer to use <code>npm</code> instead, don't worry, you can still go through the tutorial without any issues. Just add the <code>--use-npm</code> flag to the first command above and both CRA and Storybook will initialize based on this. Also while you progress through the tutorial, don't forget to adjust the commands used to their <code>npm</code> counterparts.
 </div>
 
-We can quickly check that the various environments of our application are working properly:
+After running the commands we'll need to make a small change to our app. In the root folder of the project add a new file called `.env` with the following content:
+
+```
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
Why this?

Reply:
There's a issue with babel loader that is not presently fixed. Once the reader runs the commands for create-react-app and storybook and issues `yarn test --watchAll` he's presented with:
```shell
react-scripts test
There might be a problem with the project dependency tree.
It is likely not a bug in Create React App, but something you need to fix locally.
The react-scripts package provided by Create React App requires a dependency:
  "babel-loader": "8.1.0"
Don't try to install it manually: your package manager does it automatically.
However, a different version of babel-loader was detected higher up in the tree:
  /Users/myUser/myfolder/myproject/node_modules/babel-loader (version: 8.2.1) 
Manually installing incompatible versions is known to cause hard-to-debug issues.
If you would prefer to ignore this check, add SKIP_PREFLIGHT_CHECK=true to an .env file in your project.
That will permanently disable this message but you might encounter other issues.
To fix the dependency tree, try following the steps below in the exact order:
  1. Delete package-lock.json (not package.json!) and/or yarn.lock in your project folder.
  2. Delete node_modules in your project folder.
  3. Remove "babel-loader" from dependencies and/or devDependencies in the package.json file in your project folder.
  4. Run npm install or yarn, depending on the package manager you use.
In most cases, this should be enough to fix the problem.
If this has not helped, there are a few other things you can try:
  5. If you used npm, install yarn (http://yarnpkg.com/) and repeat the above steps with it instead.
     This may help because npm has known issues with package hoisting which may get resolved in future versions.
  6. Check if /Users/myUser/myfolder/myproject/node_modules/babel-loader/node_modules/babel-loader is outside your project directory.
     For example, you might have accidentally installed something in your home folder.
  7. Try running npm ls babel-loader in your project folder.
     This will tell you which other package (apart from the expected react-scripts) installed babel-loader.
If nothing else helps, add SKIP_PREFLIGHT_CHECK=true to an .env file in your project.
That would permanently disable this preflight check in case you want to proceed anyway.
P.S. We know this message is long but please read the steps above :-) We hope you find them helpful!
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

As we don't know when someone is reading the tutorial or when this will be fixed (currently is not and i've checked if it was). 

Adding this will prevent the tutorial from breaking at the get started page.

Reply:
OK thanks for the info